applications/cheap_flights/forms.py

```diff
@@ -1,6 +1,6 @@
 from flask_wtf import FlaskForm
-from wtforms import StringField, PasswordField, SubmitField, IntegerField,SelectField, DateField
-from wtforms.validators import DataRequired, Email, Length, NumberRange, ValidationError, Optional
+from wtforms import DateField, IntegerField, SelectField, SubmitField
```

Comment:
Imports get alphabetized so you can find them easier later.  `isort` does this automatically for you.

Reply:
I see you removed the not used ones(100% on me), but .validators  was completed removed, any reason why?

#### from wtforms.validators import DataRequired, NumberRange, Optional

Reply:
Ok, I downloaded and ran the isort it didn't remove that line, probably was a mistake (LOL)

ran it for all files, looks cleaner now!

Reply:
It's possible.  I make them :P   Sorry about that.